src/gsAssembler/gsSparseSystem.h

```diff
@@ -518,7 +519,7 @@ class gsSparseSystem
     index_t unkSize(const index_t unk) const
     {return m_dims[unk];}
 
-    index_t numUnkowns() const {return m_dims.size(); }
+    index_t numUnkowns() const {return m_dims.size(); }//TODO: rename as: numUnknowns
```

Comment:
Hi Alex, you can push it, it looks great -- please fix this typo first numUnkowns-->numUnknowns